apps/src/templates/studioHomepages/CoursesTeacherEnglish.jsx

```diff
@@ -33,6 +34,9 @@ class CoursesTeacherEnglish extends Component {
     return (
       <div>
         <div>
+          {/* Currently showing CSA launch banner - May 2022 */}
+          <MarketingSkinnyBanner />
```

Comment:
Just a follow-up, if this were named CsaLaunchBanner, the comment wouldn't be necessary

Reply:
(updated the component to be CSA specific and removed comment)